src/luxonis_ml/data/utils/constants.py

```diff
@@ -0,0 +1,17 @@
+from luxonis_ml.enums import LabelType
```

Comment:
Is there a reason we import both enums differently?

Reply:
Ah I see now. Should we move them all to the same place?

Reply:
I was thinking the ones in the `data` module could just be used within the `data` module while the ones in `luxonis_ml/enums` can be used across the whole MLops stack. If you think it's cleaner to put them all in the same place I think that's fine.